mne/chpi.py

```diff
@@ -317,6 +317,14 @@ def _setup_chpi_fits(info, t_window, t_step_min, method='forward',
 
     # Set up magnetic dipole fits
     picks_meg = pick_types(info, meg=True, eeg=False, exclude=exclude)
+    if exclude == 'bads':
+        logger.debug('Static bad channels (%d): %s'
+                     % (info['bads'].__len__(), ' '.join(map('{:s}'.format,
+                                                             info['bads']))))
+    elif exclude is not None:
+        logger.debug('Static bad channels (%d): %s'
+                     % (exclude.__len__(), ' '.join(map('{:s}'.format,
+                                                        exclude))))
```

Comment:
same here about how to format. Speaking of which, this means a refactoring is in order. How about:
```
if len(exclude) > 0:
    exclude = info['bads'] if exclude == 'bads' else exclude
    logger.debug(...)
```